docs/learn/encyclopedia/data-format/xdr.mdx

```diff
@@ -16,6 +16,27 @@ Since XDR is a binary format and not as widely known as simpler formats like JSO
 
 In addition, the Horizon API server generally exposes the most important parts of the XDR data in JSON, so they are easier to parse if you are not using an SDK. The XDR data is still included (encoded as a base64 string) inside the JSON in case you need direct access to it.
 
+
+## JSON and XDR Conversion Schema
+
+:::info
+
+SDF has published libraries which support the canonical and round-trippable conversion from XDR to JSON, and from JSON to XDR, without loss of information.
+
+:::
+
+The canonical JSON and XDR Schema is defined by the [stellar_xdr crate](https://docs.rs/stellar-xdr/21.1.0/stellar_xdr/index.html) and by the [@stellar/stellar-xdr-json-web npm package](https://www.npmjs.com/package/@stellar/stellar-xdr-json-web).
+
+This conversion is visible in the Stellar CLI, Laboratory View XDR, and Hubble events table. The schema provides a round-trippable means for converting any Stellar XDR type to JSON and converting that JSON back to the identical XDR.
+
```

Comment:
We should add a note how the schema is not backwards compatible from one protocol version to a prior, and therefore it is always best to store data in XDR.

Reply:
Does [ac63218](https://github.com/stellar/stellar-docs/pull/820/commits/ac632186e70ac8662292609900709600a67b0b3b) convey this well enough, or does it need some more work?

Reply:
Looks good thanks @ElliotFriend